choice/deploy.yaml

```diff
@@ -85,7 +85,7 @@ spec:
         livenessProbe:
           failureThreshold: 10
           httpGet:
-            path: /choice2/api/v1/course
+            path: /choice2/health
```

Comment:
Even checken of dit met Spring Boot 2.x nog klopt

Reply:
Gefixt!